scalafix-core/shared/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -108,7 +108,7 @@ object Patch {
   /** A patch that does no diff/rule */
   val empty: Patch = EmptyPatch
 
-  private def merge(a: TokenPatch, b: TokenPatch): TokenPatch = (a, b) match {
+  def merge(a: TokenPatch, b: TokenPatch): TokenPatch = (a, b) match {
```

Comment:
Seeing `empty`, `merge` and no instance of `Monoid` makes me sad :(

Maybe one day :P

Reply:
haha, I know, the design is not coincidental 😉